### PR TITLE
Update dependencies to latest versions

### PR DESCRIPTION
**Pull request by merlinnot**
### Description

This PR updates all dependencies to latest versions, there are no breaking changes - some files are reformatted by Prettier.

### Code sample

Not relevant.

### CODE REVIEW

**Comment by merlinnot** [comment]
@thechenky Unfortunately I don't have permissions to assign you as a reviewer, as requested in the previous PR.

File: package.json [hunk 1]
```diff
@@ -31,36 +31,36 @@
...
   "devDependencies": {
     "@types/chai": "^4.1.7",
     "@types/chai-as-promised": "^7.1.0",
     "@types/mocha": "^5.2.7",
     "@types/mock-require": "^2.0.0",
-    "@types/nock": "^10.0.2",
-    "@types/node": "^8.10.49",
-    "@types/sinon": "^7.0.12",
+    "@types/nock": "^10.0.3",
+    "@types/node": "^12.0.10",
```
Comment (thechenky) [inline package.json]: This should still be 8, as we're not even supporting 12 now.

**Comment by thechenky** [comment]
I'm sure you're on it already, but in case you forget, remove those formatted files since you have them in the other PR.

File: package.json (hunk 1 above)
Comment (merlinnot) [reply package.json]: Nice catch, changed to `^8.10.50`.

**Comment by merlinnot** [comment]
Thanks, I removed formatted files.

**Comment by thechenky** [comment]
I'll merge this in when the tests complete.

**Pull request merged**